taipy/gui/_renderers/factory.py

```diff
@@ -360,6 +360,7 @@ class _Factory:
         .set_attributes(
             [
                 ("id",),
+                ("title",),
```

Comment:
would it be interesting to add a render dynamic boolean property ? cf. chart

Reply:
I'm not so sure what you mean here